README.md

```diff
@@ -117,6 +108,15 @@ The following properties are accepted:
 
   List of Node modules to keep out of the bundle.
 
+- `nodeBundler`
```

Comment:
I'm glad you brought that up. I did consider that approach, but I think it would introduce some complexity:

1. The [current fallback implementation](https://github.com/netlify/zip-it-and-ship-it/blob/master/src/runtimes/node/index.js#L68-L87) is quite simple, just to handle the scenario of esbuild with the zisi fallback. If we introduce an array syntax that would support any combination of bundlers in any order, things would get more complex, and I'm not sure the benefits we'd get from it justify the investment — e.g. I can't imagine a scenario where we want to be using esbuild as a fallback of zisi.

2. An array syntax for the user-facing configuration property (i.e. `functions.node_bundler` in `netlify.toml`) would be yet another point of complexity. The fact that we're asking the user to configure the bundler is already a grey line in the whole Netlify automagical approach, and throwing an array of bundlers into the mix wouldn't help simplify this, IMO.

I think we should abstract this complexity as much as possible. For example, if someone wants to use esbuild to bundle their functions, I think the instructions should be to set `node_bundler = "esbuild"` and we handle the fallback to zisi automatically, rather than asking them to do `node_bundler = "esbuild_zisi"` or `node_bundler = ["esbuild", "zisi"]`.

In any case, if we do decide that we want to have that level of control, we can always introduce an array syntax for this property without causing any breaking changes (e.g. if the value is a string, normalise it to an array with a single element).

What do you think?

(cc @ehmicky)

Reply:
Good points.
How about `bundling_mode  = "experimental"` and `bundling_mode  = "default"`?
Or something similar?
That way we abstract the underlying implementation altogether.

Reply:
Yes, I agree with @eduardoboucas. 
Using an array is a nice idea, but I am not sure it would be worth it in this specific case.